src/rebar_erlc_compiler.erl

```diff
@@ -104,7 +104,7 @@ compile(Config, Dir, OutDir) ->
     doterl_compile(Config, Dir, OutDir).
 
 -spec clean(rebar_state:t(), file:filename()) -> 'ok'.
-clean(Config, AppDir) ->
+clean(_Config, AppDir) ->
     MibFiles = rebar_utils:find_files(filename:join(AppDir, "mibs"), ?RE_PREFIX".*\\.mib\$"),
```

Comment:
Unless I'm wrong, this regex should be `?RE_PREFIX".*\\\\.mib\\\\$"` -- not that you modified any of this yourself.